src/PriceCalculatorKata/Product.cs

```diff
@@ -2,6 +2,11 @@
 
 namespace PriceCalculatorKata
 {
+    enum DiscountPrecedence
```

Comment:
each class and enum better have their own file. Please move this enum out to an Enums folder.